### DISCUSSION TITLE
Add North-Macedonia to the country list / update country list

### DISCUSSION THREAD

**Discussion by uni-t** [OpenMage/magento-lts#2117] [Ideas]
As North-Macedonia has finally declared itself independent it's time to add it to the country list of Magento.

We have also noticed that other countries or regions are also missing from the country list os it might be a tedious though usefull thing to go through the full country list and add/update countries when necessary.

**Comment by mmenozzi** [comment]
In my opinion we should stick with ISO standard here. North Macedonia is officially recognized as independent country with its own ISO code (`MK`). See [here](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).

There are also other countries officially recognized after the release of the first version of Magento 1. For example we should also add Saint Martin (DUTCH PART), ISO code `SX`.

In my opinion this should be provided with data upgrade script of the Mage_Directory module. This way upgrading an OpenMage installation to a newer version will automatically add those new countries.

**Comment by fballiano** [comment]
could you create a PR for those?

**Comment by elidrissidev** [comment]
PR submitted: #3436. North-Macedonia is still using the same ISO country code, but I updated other countries.
